aws/src/server_process/server_process.py

```diff
@@ -0,0 +1,178 @@
+import boto3
+import json
+import time
+import re
+import base64
+import socket
+import os
+
+sqs = boto3.client('sqs', region_name=os.environ['aws_region'])
+
+game_server_ip      = os.environ['aws_game_server_ip']
+game_server_port    = int(os.environ['aws_game_server_port'])
+incoming_queue_url  = os.environ['aws_incoming_queue_url']
+outgoing_queue_url  = os.environ['aws_outgoing_queue_url']
+
+def decrypt_password(ciphertext_base64):
+    kms_client = boto3.client('kms', region_name=os.environ["aws_region"])
```

Comment:
Consider moving this to the top of the file so that all of the environment variables are together.  Easier not to miss one that way.